z2/resources/start_node.tera.sh

```diff
@@ -8,17 +8,19 @@ NODE_PRIVATE_KEY=""
 CONFIG_FILE="${CHAIN_NAME}.toml"
 CHECKPOINT_FILE=""  # The optional checkpoint file to mount, if provided.
 DATA_FOLDER=$(pwd)/data
+SCILLA_SERVER_PORT=3111
```

Comment:
FWIW I'd pick something between 10,000 and 65,000 if we want collisions to be most unlikely.